frontends/election-manager/src/components/hand_marked_paper_ballot.tsx

```diff
@@ -493,25 +521,21 @@ export function CandidateContestChoices({
           </BubbleMark>
         </Text>
       ))}
-      {writeInCandidates?.map((candidate) => (
-        <Text key={candidate.name} bold noWrap>
-          <BubbleMark position={targetMarkPosition} checked>
-            <span>
-              <strong>{candidate.name}</strong> (
-              {dualLanguageWithSlash('write-in')})
-            </span>
-          </BubbleMark>
-        </Text>
-      ))}
       {contest.allowWriteIns &&
-        remainingChoices.map((k) => (
-          <WriteInItem key={k} data-write-in>
-            <BubbleMark position={targetMarkPosition}>
+        writeInItemKeys.map((key, position) => (
+          <WriteInItem key={key} data-write-in>
```

Comment:
Where is the `data-write-in` attribute used?

Reply:
I do not know. Nowhere, as far as I can tell? Same goes for other data labels, like `data-candidate` in this file.